src/NuGetGallery/Services/UserService.cs

```
@@ -151,5 +157,52 @@ public virtual User FindByUsername(string username)
+        {
+            accountToTransform = accountToTransform ?? throw new ArgumentNullException(nameof(accountToTransform));
+            adminUser = adminUser?? throw new ArgumentNullException(nameof(adminUser));
+            
+            if (string.IsNullOrWhiteSpace(token))
+            {
+                throw new ArgumentNullException(nameof(token));
+            }
+
+            var tenantId = adminUser.GetTenantId();
+            if (string.IsNullOrWhiteSpace(tenantId))
+            {
+                // todo: add security policy to organization below to enforce this (future work, with manage organization)
+                throw new TransformAccountException(Strings.TransformAccount_AdminDoesNotHaveTenantId);
+            }
+            
+            try
+            {
+                var database = EntitiesContext.GetDatabase();
+                var result = await database.ExecuteSqlCommandAsync(
+                    ExecMigrateToOrganization,
+                    new SqlParameter("orgKey", accountToTransform.Key),
+                    new SqlParameter("adminKey", adminUser.Key),
+                    new SqlParameter("token", token)
+                    );
+
+                // For ExecuteSqlCommandAsync result, see SqlDataReader.RecordsAffected.
+                // Result was -1 (found no migration requests with select) or 0 (no insert, update or delete).
+                if (result <= 0)
+                {
+                    // Stored procedure check failed (i.e., migration request didn't match, or membership existed).
+                    throw new TransformAccountException(Strings.TransformAccount_DatabaseError);
+                }
+            }
+            catch (Exception ex) when (ex is SqlException || ex is DataException)
+            {
+                // EF exception when saving account transformation to the database.
+                throw new TransformAccountException(Strings.TransformAccount_DatabaseError);
```

The exceptions you throw here are swallowed in UsersController, so we have no log of them. Please add a trace (somewhere)